### PR TITLE
SubS Actor Getters OK

### PR DESCRIPTION
**Pull request by hensldm**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

The first set of decomped code in SubS is the actor getters. These four functions return an actor based on some condition.
I also noticed that the rename_syms bash script did not update the size csvs, so added that in as well.

### CODE REVIEW

File: src/code/z_sub_s.c [hunk 1]
```diff
@@ -100,7 +158,21 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013E5CC.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013E640.s")
+Actor* SubS_GetActorByFunction(GlobalContext* globalCtx, Actor* actor, Actor* actorListStart, u8 actorCategory,
```
Comment (AngheloAlf) [inline src/code/z_sub_s.c:161]: I'm not completely convinced with the `ByFunction` suffix. `ByCallback` may be better, but I'm not sure either. 
Would be nice to see what everybody else thinks about this.

**Review by AngheloAlf — COMMENTED** [review]
Nice work here.
Just a minor comment with the name of a function

File: src/code/z_sub_s.c (hunk 1 above)
Comment (hensldm) [reply src/code/z_sub_s.c:161]: What about something like `ByVerification` to go along with how the function argument is called `verifyActor`?

Comment (EllipticEllipsis) [reply src/code/z_sub_s.c:161]: This function's pretty weird... it only seems to be used twice in the codebase: once in EnTk, with a verifier that always returns false, and once in ObjHunsui,
```c
? func_80B9D288(s32 arg0, ? arg1, void *arg2, ? arg3) {
    ? phi_v1;

    phi_v1 = 0;
    if ((arg2->unk1C & 0xF) == 8) {
        phi_v1 = 1;
    }
    return phi_v1;
}
```
i.e. the bottm nybble of params is 8.

Comment (hensldm) [reply src/code/z_sub_s.c]: Note that I also changed the other names as well to follow the same format. Let me know if they are still good enough.

**Review by AngheloAlf — APPROVED** [review]

**Comment by hensldm** [comment]
Fixed the conflict.

File: src/code/z_sub_s.c [hunk 2]
```diff
@@ -68,7 +135,22 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013D924.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_ActorCategoryIterateById.s")
+/**
+ * Finds the first actor instance of a specified Id and category.
+ */
+Actor* SubS_ActorFind(GlobalContext* globalCtx, Actor* actorListStart, u8 actorCategory, s16 actorId) {
```
Comment (engineer124) [inline src/code/z_sub_s.c:141]: Have you considered swapping the order of the names of these functions? Something like:
`SubS_FindActor`, `SubS_FindNearestActor`, `SubS_FindActorCustom`, and `SubS_FindDoor`? Just a thought, either is fine with me

**Review by engineer124 — APPROVED** [review]
LGTM, just one optional comment

File: src/code/z_sub_s.c (hunk 2 above)
Comment (hensldm) [reply src/code/z_sub_s.c:141]: Those do seem better so going to go with those.

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**